spark-doris-connector/src/main/java/org/apache/doris/spark/load/DorisStreamLoad.java

```diff
@@ -218,7 +221,7 @@ public long load(Iterator<InternalRow> rows, StructType schema)
             this.loadUrlStr = loadUrlStr;
             HttpPut httpPut = getHttpPut(label, loadUrlStr, enable2PC, schema);
             RecordBatchInputStream recodeBatchInputStream = new RecordBatchInputStream(RecordBatch.newBuilder(rows)
-                    .format(fileType)
+                    .format(dataFormat)
```

Comment:
Currently, the format of StreamLoad is lowercase in the Header. Can uppercase transmission also work?